time/chrono.h

```diff
@@ -28,20 +28,52 @@ SOFTWARE.
 #define BRICKS_TIME_CHRONO_H
 
 #include <algorithm>
+#include <thread>
 #include <chrono>
 
 #include "../port.h"
 #include "../strings/fixed_size_serializer.h"
 
 namespace bricks {
-
 namespace time {
 
 enum class EPOCH_MILLISECONDS : uint64_t {};
 enum class MILLISECONDS_INTERVAL : uint64_t {};
 
+}  // namespace bricks::time
+}  // namespace bricks
+
+inline bricks::time::MILLISECONDS_INTERVAL operator-(bricks::time::EPOCH_MILLISECONDS lhs,
```

Comment:
I see that it is not a new code. This question worries me till last year: why do you want to strictly separate uint64_t from EPOCH_MILLISECOND if it is still exactly the same arithmetics and context? Your approach is bloating the code and makes it hardly understandable than simple typedef uint64_t EPOCH_MILLISECONDS;